.github/workflows/integration-cml.yml

```diff
@@ -226,20 +230,26 @@ jobs:
         shell: bash
         run: python -c "from pylibsshext import __full_version__; print(__full_version__)"
 
-      - name: Run ansible-test network-integration - NETCONF tests
-        run: ansible-test network-integration --inventory "${{ steps.render_inventory.outputs.inventory_path }}" netconf_config netconf_get netconf_rpc -vvvv
-        working-directory: ${{ steps.identify.outputs.collection_path }}
-        env:
-          LAB_NODES: ${{ needs.lab-create.outputs.lab_nodes_json }}
+      # - name: Run ansible-test network-integration - NETCONF tests
+      #   run: ansible-test network-integration --inventory "${{ steps.render_inventory.outputs.inventory_path }}" netconf_config netconf_get netconf_rpc -vvvv
```

Comment:
If we are not reenabling them , can ew remove this.

Reply:
sure, I was keeping them incase this PR gives hard time in testing,.